src/Backend/Core/Engine/Base/AjaxAction.php

```diff
@@ -55,19 +52,14 @@ public function getContent()
     /**
      * Output an answer to the browser
      *
-     * @param int    $statusCode The status code for the response, use the
+     * @param int $statusCode The status code for the response, use the
      *                           available constants:
      *                           self::OK, self::BAD_REQUEST, self::FORBIDDEN, self::ERROR
-     * @param mixed  $data       The data to output.
-     * @param string $message    The text-message to send.
+     * @param mixed $data The data to output.
+     * @param string $message The text-message to send.
      */
-    public function output($statusCode, $data = null, $message = null)
+    public function output(int $statusCode, $data = null, string $message = null)
     {
-        $statusCode = (int) $statusCode;
-        if ($message !== null) {
-            $message = (string) $message;
-        }
-
         $response = array('code' => $statusCode, 'data' => $data, 'message' => $message);
```

Comment:
yea, was going to do a code style clean up at the end as well, that way I can automatically convert all the arrays